addons/website/models/website_page.py

```diff
@@ -208,6 +208,11 @@ def get_website_meta(self):
         self.ensure_one()
         return self.view_id.get_website_meta()
 
+    def _can_be_cached(self, response):
+        """ return False if at list one blacklisted's word are present in content """
+        blacklist = ('data-snippet="s_website_form"', 'data-no-page-cache=', )
+        return not any(black in str(response) for black in blacklist)
```

Comment:
Shouldn't we create an util method to get the blacklist words, so it can be overridden in other modules?
eg `_get_cache_blacklist_words()`

Reply:
Agreed :+1: